docs/TOC.yml

```diff
@@ -43,10 +43,14 @@
               href: ios/pair-to-mac.md
             - name: Remote iOS simulator for Windows
               href: ios/remote-simulator.md
+            - name: Xcode Sync
```

Comment:
I don't think Xcode sync should be mentioned under Getting Started. Getting started is meant to be basic stuff useful for building your first app.

Reply:
ah I meant to add this where the content for Mac cat was added- I'll adjust it!